charts/argo-cd/templates/argocd-repo-server/clusterrole.yaml

```diff
@@ -1,23 +1,10 @@
-{{- if .Values.createClusterRoles }}
+{{- if and .Values.createClusterRoles .Values.repoServer.clusterRoleRules.enabled }}
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ include "argo-cd.repoServer.fullname" . }}
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.repoServer.name "name" .Values.repoServer.name) | nindent 4 }}
 rules:
-  {{- if .Values.repoServer.clusterRoleRules.enabled }}
-    {{- toYaml .Values.repoServer.clusterRoleRules.rules | nindent 2 }}
-  {{- else }}
-  - apiGroups:
```

Comment:
I'm not sure if we want to put admin API groups in values.yaml as default or not. WDYT?